x-pack/plugins/siem/public/components/charts/areachart.test.tsx

```diff
@@ -38,16 +38,8 @@ describe('AreaChartBaseComponent', () => {
       wrapper = mount(<AreaChartBaseComponent height={100} width={120} data={mockAreaChartData} />);
     });
 
-    it('should render two area series', () => {
-      expect(wrapper.find('EuiAreaSeries')).toHaveLength(2);
-    });
-
-    it('should render a customized x-asix', () => {
-      expect(wrapper.find('EuiXAxis')).toHaveLength(1);
-    });
-
-    it('should render a customized y-asix', () => {
-      expect(wrapper.find('EuiYAxis')).toHaveLength(1);
+    it('should render Chart', () => {
+      expect(wrapper.find('Chart')).toHaveLength(1);
```

Comment:
Removing further testing for charts' detail here because
1. It is render by canvas so not able to be tested from outside of the library. - https://github.com/elastic/elastic-charts/issues/215
2. Most of the cases have been tested inside elastic-charts.